qiita_pet/templates/artifact_ajax/processing_artifact.html

```diff
@@ -292,33 +376,35 @@
       }
       sel_artifacts_info[node.id] = {'type': node.type, 'name': node.name}
     }
-    $.get('{% raw qiita_config.portal_dir %}/study/process/commands/', {artifact_types: types})
+    $.get('{% raw qiita_config.portal_dir %}/study/process/commands/', {artifact_types: types, exclude_analysis: !is_analysis_pipeline})
```

Comment:
the use of the negative "exclude" makes this confusing, would it be better to refactor and describe it as `include_analysis`?